controllers/appgroup_reconciler.go

```diff
@@ -99,7 +99,22 @@ func (r *ApplicationGroupReconciler) reconcileApplications(l logr.Logger, appGro
 		}
 
 		if appCh.Dependencies() != nil {
+			isImplicitSubchartsList := false
+			// Update the application spec with a list of subcharts with no inter-dependence
+			if len(application.Spec.Subcharts) == 0 {
```

Comment:
What happens if we specify some subcharts (but not all) and we want the ones not specified to not wait?

Reply:
Also, can you add a testcase for this scenario?

Reply:
Good point - I will add handlers for that case. Sure - will add testcases !

Reply:
@jonathan-innis I made the change and verified it manually. We still need to add tests for this scenario but I will track that in a separate issue since we need this to cut our next release. Let me know if that works and then I can create the other test issue